libs/api/v1/src/lib/api-v1.ts

```diff
@@ -1,34 +1,45 @@
 import FirecrawlApp from '@mendable/firecrawl-js'
 import { generateQuiz, getResult, getTranscript, uploadFile, waitUntilJobIsDone } from '@nmit-coursition/ai'
-import { reportUsage, validateApiKey } from '@nmit-coursition/api/utils'
+import { formatApiErrorResponse, reportUsage, validateApiKey } from '@nmit-coursition/api/utils'
+import { type ApiErrorCode, ERROR_LIST } from '@nmit-coursition/api/utils'
 import {
   allowedDeepgramLanguagesAsType,
   allowedLlamaParseLanguagesAsType,
   languages,
   languagesAsType,
 } from '@nmit-coursition/utils'
 import { Elysia, t } from 'elysia'
-import { type ApiErrorCode, ERROR_LIST } from '../../../utils/src/errorList'
+
+const errorResponseType = t.Object({
```

Comment:
**suggestion:** As I see repeated type in `api/v1`  and `api/dev`, I think it is confirmed, that we should move this into `api/utils`.